src/pool/srv_pool.c

```diff
@@ -115,6 +115,15 @@ reconf_end(struct pool_svc_reconf *reconf)
 	reconf->psc_canceled = false;
 }
 
+static void
+reconf_wait(struct pool_svc_reconf *reconf)
+{
+	ABT_mutex_lock(reconf->psc_mutex);
+	while (reconf->psc_in_progress)
+		ABT_cond_wait(reconf->psc_cv, reconf->psc_mutex);
```

Comment:
How long the wait may take? It is possible that "dmg check stop" coming in during such wait, we may need some interface to wakeup the waiting reconf under such case to avoid being blocked for very long time?

Reply:
I think the simplest way is to just do not wait for the reconf, like we don't wait for the automatic reconfs. Because the PS is already available and the pool can already work. Tests that must verify the reconf result can just wait for a minute or so.

Reply:
What will happen if pool stop during reconfs? Because the user may wants to restart daos check before reconfs complete, right?

Reply:
If some code calls ds_pool_svc_stop, and if this is the PS leader with a running reconf ULT, then the reconf ULT will be canceled. I think the PS reconf is not part of CR and the CR state of this pool should move to `DONE` regardless of the PS reconf.